### ISSUE TITLE
Provide the option to import a snippet into another snippet using the Libraries tab

### ISSUE THREAD

**Issue by jakobpn** [OfficeDev/script-lab#656]
This would enable splitting up a larger snippet into smaller chunks and also the ability to reuse code across snippets.

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used:
* Browser Name and version:
* Operating System and version (desktop or mobile):

## Useful logs

- [ ] Fiddler traces
- [ ] Console errors
- [ ] Screenshots
- [ ] Clip or GIFs

**Comment by Zlatkovsky** [comment]
I agree it would be a cool feature, and it could be useful for a variety of stuff (including dialog API).
What is less clear to me is how you'd share such snippet (gist/clipboard), or what happens when you duplicate the snippet.  The referencing can get hard to manage.  If we really want multiple chunks, I wonder if perhaps we can support multiple script/html/css "files" within the same snippet.

We can chat more about it in person.
